README.md

```diff
@@ -30,6 +30,40 @@ The benchmarks that are here by default are made to test the performance of Phar
 2. [Optional] Add a `.sh` file in the `/vms` folder with the instructions to download your vm. If not you can use one of the already present vms.
 3. Execute `runBenchs.sh name_of_your_benchmark` with the name.s of your benchmark.s
 
+# Create your own configuration
+
+You can create your own benchmark configuration by creating a `.sh` file using the following template:
+
+```bash
+#!/bin/bash
+
+IMAGES="<the images you want your benchmark to run in>"
+VMs="<the VMs you want your benchmark to run with>"
+PHARO_CMD="<the command you want the VM to execute>"
+
+source "$1/bench.inc"
+```
+
+and then add it to the `/benchs` folder.
+
+You can also chain the following lines to run the same command with other images and/or VMs:
+
+```bash
+#!/bin/bash
+
+IMAGES="<the images you want your benchmark to run in>"
+VMs="<the VMs you want your benchmark to run with>"
+PHARO_CMD="<the command you want the VM to execute>"
+
+source "$1/bench.inc"
+IMAGES="<other images>"
+VMs="<other VMs>"
+
+source "$1/bench.inc"
+```
```

Comment:
```suggestion
```bash
#!/bin/bash

# Command
PHARO_CMD="<the command you want the VM to execute>"

# First configuration
IMAGES="<the images you want your benchmark to run in>"
VMs="<the VMs you want your benchmark to run with>"

# Run
source "$1/bench.inc"


# Second configuration
IMAGES="<other images>"
VMs="<other VMs>"

# Run
source "$1/bench.inc"
```
```

Reply:
like it

Reply:
+1